src/router/index.js

```diff
@@ -0,0 +1,66 @@
+import VueRouter from "vue-router";
+
+import HomePage from "@/components/pages/public/HomePage.vue";
+
+export const routes = [
+  {
+    path: "/",
+    name: "Home",
+    component: HomePage,
+  },
+  {
+    path: "/screenings",
+    name: "Screenings",
+    component: () => import("@/components/pages/public/ScreeningsPage.vue"),
+  },
+  {
+    path: "/movies",
+    name: "Movies",
+    component: () => import("@/components/pages/public/AllMoviesPage.vue"),
+  },
+  {
+    path: "/movies/:movieId",
+    name: "SingleMovie",
+    component: () => import("@/components/pages/public/MovieDetailPage.vue"),
+  },
+  {
+    path: "/contact",
+    name: "Contact",
+    component: () => import("@/components/pages/public/ContactPage.vue"),
+  },
+  {
+    path: "/login",
+    name: "Login",
+    component: () => import("@/components/pages/public/LoginPage.vue"),
+  },
+  {
+    path: "/register",
+    name: "Register",
+    component: () => import("@/components/pages/public/RegisterPage.vue"),
+    children: [
+      {
+        path: "/register/first",
```

Comment:
We should omit the "parent" part of the path 😄 
```suggestion
        path: "/first",
```